t/plugin/limit-count.t

```diff
@@ -637,3 +637,55 @@ GET /t
 passed
 --- no_error_log
 [error]
+
+
+
+=== TEST 21: when the count is changed, check  the limit is correct
```

Comment:
need add test case to cover: `count` from 1 to 2.

Reply:
and `count` from 2 to 2.

Reply:
added